README.md

```diff
@@ -10,27 +10,31 @@ Built with [React](https://facebook.github.io/react/) and [Redux](https://github
 The backend is supported by Docker. If you do not have it already, you will need to install it.
 * For OS X, go [here](https://docs.docker.com/docker-for-mac/).
 * For Windows, go [here](https://docs.docker.com/docker-for-windows/).
-* For Ubuntu and other Linux distributions, go [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
+* For Ubuntu and other Linux distributions, install
+[docker](https://docs.docker.com/engine/installation/linux/ubuntulinux/) and
+[docker-compose](https://docs.docker.com/compose/install/).
+
+To [avoid having to use sudo when you use the docker command]
+(https://docs.docker.com/engine/installation/linux/ubuntulinux/#/create-a-docker-group),
+create a Unix group called docker and add users to it:
+> 1. `sudo groupadd docker`
+> 2. `sudo usermod -aG docker $USER`.
 
 Once installed, start the Docker application (if on a Mac), then go to the project directory and run:
 
-1. `docker-compose up`
+1. `docker-compose up -d`
```

Comment:
Yes, -d for detach from terminal.
Without that, I was staring, wondering if startup was done, then I hit ctrl-C, and it stopped the containers,

After detach, you can reattach with `docker attach --sig-proxy=false container_name`
or access the container with a new shell `docker exec -it container_name /bin/bash`